File: 100-Days-Series/Day 18/Count_Vowels_And_Consonants.py

def getCountVowels(str):
    countVowels=0
    for i in range(0,len(str),1):
        ch=str[i]
        if ch=='a' or ch=='e' or ch=='i' or ch== 'o' or ch=='u' or ch=='A' or ch=='E' or ch=='I' or ch== 'O' or ch=='U':
            countVowels+=1
    return countVowels

def getCountConsonants(str):
    countConsonants=0
    for i in range(0,len(str),1):
        ch=str[i]
        if ch!='a' or ch!='e' or ch!='i' or ch!= 'o' or ch!='u' or ch!='A' or ch!='E' or ch!='I' or ch!= 'O' or ch!='U':
            countConsonants+=1
    return countConsonants

string=input("Enter a String : ")
countVowels=getCountVowels(string)
countConsonants=getCountConsonants(string)
print(string,' in this String...')
print('Total Vowels are : ',countVowels)
print('Total Consonants are : ',countConsonants)

File: 100-Days-Series/Day 18/Count_Vowels_And_Consonants.cpp
#include<iostream>
#include<string>
using namespace std;
// Function for count Vowels
int getCountVowels(string str){
    int countVowels=0;
    int len=str.length();
    for(int i=0;i<len;i++){
        char ch=str[i];
        if(ch=='a' || ch=='e' || ch=='i' || ch=='o' || ch=='u' ||ch=='A' || ch=='E' || ch=='I' || ch=='O' || ch=='U'){
            countVowels++;
        }
    }
    return countVowels;
}
// Function for count Consonants
int getCountConsnants(string str){
    int countConsonants=0;
    int len=str.length();
    for(int i=0;i<len;i++){
        char ch=str[i];
        if(!(ch=='a' || ch=='e' || ch=='i' || ch=='o' || ch=='u' ||ch=='A' || ch=='E' || ch=='I' || ch=='O' || ch=='U')){
            countConsonants++;
        }
    }
    return countConsonants;
}
// Let's Start here
int main(){
    string str;
    cout<<"\nEnter a String : ";
    cin>>str;
    cout<<"\n'"<<str<<"' in this String...";
    int totalVowels,totalConsonants;
    totalVowels=getCountVowels(str);
    totalConsonants=getCountConsnants(str);
    cout<<"\nTotal Vowels are : "<<totalVowels;
    cout<<"\nTotal Consonants are : "<<totalConsonants;
    }